vumi/middleware/tests/test_base.py

```diff
@@ -280,9 +288,12 @@ def test_parse_yaml(self):
 
     @inlineCallbacks
     def test_sort_by_priority(self):
-        priority2 = ToyMiddleware('priority2', {"priority": 2}, self)
-        priority1_1 = ToyMiddleware('priority1_1', {"priority": 1}, self)
-        priority1_2 = ToyMiddleware('priority1_2', {"priority": 1}, self)
+        priority2 = ToyMiddleware('priority2', {}, self)
```

Comment:
Ah! I see this is just for testing `_sort_by_priority`. Happy with it the way it is.